### PR TITLE
grc: Updated flow_graph.py.mako with import statement (maint-3.8)

### LINKED ISSUE #3141: Print function used without __future__ import
The print function (with keyword argument) is used in [flow_graph.py.mako](https://github.com/gnuradio/gnuradio/blob/maint-3.8/grc/core/generator/flow_graph.py.mako#L250) without `from __future__ import print_function`

Inserting the import after line 8 appears to work: a quick skim of the template did not show anywhere the print statement was used to print a tuple or where the parenthesis were missing (and if there is, it would need fixing for 3.9 anyway), but I have not thoroughly tested the fix. Users cannot simply use an import block because future imports must come before any others.

### PR DESCRIPTION
**Pull request by alekhgupta1441**
Fixes #3141 
Added:
 ~~~~
from __future__ import print_function

### CODE REVIEW

File: grc/core/generator/flow_graph.py.mako [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <%
 from sys import version_info
 from gnuradio import eng_notation
+from __future__ import print_function
```
Comment (mormj) [inline grc/core/generator/flow_graph.py.mako:5]: doesn't the __future__ import need to come before any other imports?

Comment (alekhgupta1441) [reply grc/core/generator/flow_graph.py.mako:5]: Yes, it should be at the top. Now corrected.

**Comment by mormj** [comment]
@alekhgupta1441 - thank you for updating the commit.  As a good general practice, can you also please squash your commits since the first one is covered by the second one.  To do this, just go into your working directory and:
```
git rebase -i HEAD~2
# for this case you can change "pick" to "drop" for the first commit
# you can also change the second commit to "squash" and then update the commit message
git push --force origin branchname
```

**Comment by alekhgupta1441** [comment]
@mormj Rebased finally!

**Comment by drmpeg** [comment]
This commit breaks Python3. Here's the error message when loading a flow graph.
```
ubuntu@arm:~/xfer$ gnuradio-companion
Traceback (most recent call last):
  File "/opt/gnuradio-3.9.0git/bin/gnuradio-companion", line 102, in <module>
    run_main()
  File "/opt/gnuradio-3.9.0git/bin/gnuradio-companion", line 95, in run_main
    exit(main())
  File "/opt/gnuradio-3.9.0git/lib/python3/dist-packages/gnuradio/grc/main.py", line 73, in main
    from .gui.Platform import Platform
  File "/opt/gnuradio-3.9.0git/lib/python3/dist-packages/gnuradio/grc/gui/Platform.py", line 27, in <module>
    from ..core.platform import Platform as CorePlatform
  File "/opt/gnuradio-3.9.0git/lib/python3/dist-packages/gnuradio/grc/core/platform.py", line 39, in <module>
    from .generator import Generator
  File "/opt/gnuradio-3.9.0git/lib/python3/dist-packages/gnuradio/grc/core/generator/__init__.py", line 19, in <module>
    from .Generator import Generator
  File "/opt/gnuradio-3.9.0git/lib/python3/dist-packages/gnuradio/grc/core/generator/Generator.py", line 25, in <module>
    from .hier_block import HierBlockGenerator, QtHierBlockGenerator
  File "/opt/gnuradio-3.9.0git/lib/python3/dist-packages/gnuradio/grc/core/generator/hier_block.py", line 7, in <module>
    from .top_block import TopBlockGenerator
  File "/opt/gnuradio-3.9.0git/lib/python3/dist-packages/gnuradio/grc/core/generator/top_block.py", line 18, in <module>
    python_template = Template(filename=PYTHON_TEMPLATE)
  File "/usr/lib/python3/dist-packages/mako/template.py", line 338, in __init__
    module = self._compile_from_file(path, filename)
  File "/usr/lib/python3/dist-packages/mako/template.py", line 416, in _compile_from_file
    filename)
  File "/usr/lib/python3/dist-packages/mako/template.py", line 711, in _compile_text
    code = compile(source, cid, 'exec')
  File "_opt_gnuradio_3_9_0git_lib_python3_dist_packages_gnuradio_grc_core_generator_flow_graph_py_mako", line 42
SyntaxError: from __future__ imports must occur at the beginning of the file
```

**Comment by drmpeg** [comment]
@mbr0wn Bump.

**Comment by argilo** [comment]
I encountered the same problem as @drmpeg and bisected it to this commit.

**Comment by argilo** [comment]
I opened #3508 to fix the bug this pull request caused.

**Comment by dkozel** [comment]
Thanks @argilo and @drmpeg. I'll go attend to the other PR to get this fixed, but then we need to figure out why the CI didn't catch this.

**Comment by argilo** [comment]
@dkozel `grcc` fails in exactly the same way as `gnuradio-companion`, so just running `grcc` once during the CI run would probably be a good way to catch bugs like this.

**Comment by argilo** [comment]
A good next step might be to use `grcc` to compile all the `.grc` files in `/*/examples/` and verify that the output is at least syntactically valid Python. That could catch bugs in the block definitions.

**Pull request merged**